iml-manager-cli/Cargo.toml

```diff
@@ -24,6 +24,7 @@ tokio = { version = "0.2", features = ["macros"] }
 tracing = "0.1"
 tracing-subscriber = "0.1"
 number-formatter = { path = "../number-formatter", version = "0.1" }
+clt = "0.0.6"
```

Comment:
I don't like adding a dependency that's wasn't updated for more than a year and isn't maintained at this point.

Given that there's just one function needed, I propose implementing it in our code.

Reply:
We'll probably need more of these in the future. 

I'd suggest using https://docs.rs/dialoguer/0.5.0/dialoguer/ instead, which similar to `console` does not have a dependency on termion (it actually depends on `console`) which means we can still compile on windows.

Reply:
Ok, I'll switch to dialoguer.